fbt_enforcement/mailer.py

```diff
@@ -0,0 +1,110 @@
+from reportlab.lib.pagesizes import letter
+from reportlab.lib.styles import getSampleStyleSheet, ParagraphStyle
+from reportlab.platypus import SimpleDocTemplate, PageTemplate, Frame, Paragraph, Spacer, Image
+from reportlab.lib.units import inch
+
+
+def create_speeding_ticket_pdf(car_data):
+    # Content
+    file_name = 'traffic_citation.pdf'
+    document_title = 'Traffic Citation'
+    title = 'Notice of Speed Violation'
+    border_line = ['_______________________________________________________']
+
+    citation_details = [
+        f'Location: {car_data["owner"]["address"]}',
+        # f'Date: {car_data["date"]}',  # Add the appropriate date field from your data
+        # f'Time: {car_data["time"]}',  # Add the appropriate time field from your data
+        # f'Sign Speed: {car_data["sign_speed"]} MPH',
+        f'Vehicle Speed: {car_data["miles_per_hour"]} MPH',
+        f'Plate Number: {car_data["plate_number"]}',
+        f'Vehicle Make: {car_data["vehicle"]["make"]}'
+    ]
+    citation_body = [
+        '',  # Empty string for a line break
+        'Your vehicle was photographed speeding, '
+        'in violation of 7-15 and 8-1-2-6, Eugene Ordinances, '
+        'and OSL 1972 66-7-104 of the Oregon State Motor Vehicle Code',
+        '',  # Empty string for a line break
+        'As the vehicle\'s registered owner, you are liable for the violation. '
+        'The civil penalty is $100.00 (payment instructions below). '
+        'Payment is deemed an admission and waiver of your right to appeal. '
+        'Failure to pay may result in this case being forwarded to a collection company.',
+        '',  # Empty string for a line break
+        'PAYMENT OF THE PENALTY AMOUNT FOR THE VIOLATION WILL NOT RESULT IN '
+        'POINTS AND CANNOT BE USED TO INCREASE YOUR INSURANCE RATES.',
+        '',  # Empty string for a line break
+        'City of Eugene Officer\'s Certificate',
+        'Based on personal inspection of vehicle images showing '
+        'violation of 7-15 and 8-1-2-6, Eugene Ordinances, '
+        'and OSL 1972 66-7-104 of the Oregon State Motor Vehicle Code',
+        '',  # Empty string for a line break
+    ]
+    officer_signature = [
+        'Sworn to or affirmed by:',
+        '________________________________',
+        'Signature of Officer       Date'
+    ]
```

Comment:
Any kind of static information like this should be on the page already. We should have a "traffic ticket" pdf. Then you basically just slap the data where it needs to go. This is much easier than creating an entire PDF with all the added elements and trying to format every little bit. 
![image](https://github.com/Mannuello/fbt-enforcement/assets/30988215/8f9b3cb2-a996-469a-9eda-a7ceef86d9d7)